source/NVDAObjects/IAccessible/sysListView32.py

```diff
@@ -95,7 +95,7 @@ class LVITEM(Structure):  # noqa: F405
 		("iSubItem", c_int),  # noqa: F405
 		("state", c_uint),  # noqa: F405
 		("stateMask", c_uint),  # noqa: F405
-		("pszText", c_void_p),  # noqa: F405
+		("pszText", c_ulong),  # noqa: F405
```

Comment:
Please add appropriate comments to explain why it is necessary to diverge from `commctrl.h` definitions here
```suggestion
		# A pointer to a buffer containing the text of the item.
		# Note that the pointer size is dictated by the process that hosts the list item, not the process that fetches the list item information.
		("pszText", c_ulong),  # noqa: F405
```